### PR TITLE
added sprint project to showcase

### PR DESCRIPTION
**Pull request by ketanbj**
Added a generated background and text description

### CODE REVIEW

File: src/content.json [hunk 1]
```diff
@@ -46,6 +46,35 @@
...
+		"cover": "sprint.png",
+		"logos": [
+			{
+				"image": "https://kokkos.org/img/kokkos-logo.png",
+				"alt": "The Kokkos C++ Performance Portability Ecosystem is a production level solution for writing modern C++ applications in a hardware agnostic way.",
+				"url": "https://kokkos.org/"
+			},
+			{
+				"image": "https://www.lammps.org/movies/logo.gif",
+				"alt": "The Kokkos C++ Performance Portability Ecosystem is a production level solution for writing modern C++ applications in a hardware agnostic way.",
```
Comment (krachwal) [inline src/content.json:62]: This alt text is the same as the Kokkos. Update or remove.

File: src/content.json [hunk 2]
```diff
@@ -46,6 +46,35 @@
 		"start_date": "November 2023",
 		"end_date": "May 2024"
 	},
+	{
+		"title": "Kokkos Accelerated Machine Learned Interatomic Potential",
+		"description": "During this engagement, we evaluated the performance characteristics of simulations based on <a href=https://arxiv.org/abs/2401.00096 target=_blank>foundational model</a> for machine learned inter-atomic potential jointly proposed by a team of 60 scientists . ",
+		"background": "sprint10.png",
+		"cover": "sprint.png",
```
Comment (davidbrownell) [inline src/content.json:53]: I don't see this file checked in.

File: src/content.json [hunk 3]
```diff
@@ -46,6 +46,35 @@
 		"start_date": "November 2023",
 		"end_date": "May 2024"
 	},
+	{
+		"title": "Kokkos Accelerated Machine Learned Interatomic Potential",
+		"description": "During this engagement, we evaluated the performance characteristics of simulations based on <a href=https://arxiv.org/abs/2401.00096 target=_blank>foundational model</a> for machine learned inter-atomic potential jointly proposed by a team of 60 scientists . ",
+		"background": "sprint10.png",
```
Comment (davidbrownell) [inline src/content.json:52]: Can you give these image filename a more descriptive name (maybe something that includes `kokkos`).

**Review by davidbrownell — CHANGES_REQUESTED** [review]

File: src/content.json (hunk 3 above)
Comment (ketanbj) [reply src/content.json]: Changed the name of the file.

**Review by davidbrownell — DISMISSED** [review]

**Pull request closed without merging**